src/dev/impl/DevToys/ViewModels/Tools/Graphic/PngJpgCompressor/PngJpgCompressorToolViewModel.cs

```diff
@@ -133,9 +133,7 @@ private async Task ExecuteSelectFilesAreaDragDropCommandAsync(DragEventArgs? par
                     IStorageItem storageItem = storageItems[i];
                     if (storageItem is StorageFile file)
                     {
-                        string fileExtension = Path.GetExtension(file.Name);
-
-                        if (SupportedFileExtensions.Any(ext => string.Equals(ext, fileExtension, StringComparison.OrdinalIgnoreCase)))
+                        if (SupportedFileExtensions.Any(ext => string.Equals(ext, Path.GetExtension(file.Name), StringComparison.OrdinalIgnoreCase)))
```

Comment:
I think what @btiteux was suggesting is to use `file.FileType` instead of `Path.GetExtension`.